doc/faq.rst

```diff
@@ -26,6 +26,27 @@ See :ref:`contributing`. Before wanting to add a new algorithm, which is
 usually a major and lengthy undertaking, it is recommended to start with :ref:`known
 issues <easy_issues>`.
 
+What's the best way to ask questions about scikit-learn?
+--------------------------------------------------------------
+The quickest way to get replies to your questions is posting on 
+`Stack Overflow <http://stackoverflow.com>`_ or 
```

Comment:
add "(about scikit-learn usage)" vs "(about machine learning techniques)" on the next line